docs/library/jesd204/xgt_wizard/index.rst

```diff
@@ -0,0 +1,382 @@
+
+This call makes 4 instances of transceivers. Now you can double click on the <ip_name>.xci 
+from the sources window to further customize the IP, including configurations where RX 
+and TX have different rates. After you are all set, run the Parsing_script to get the 
+list of parameters that need to be changed into the project.
+
+get_diff_params
+********************************************************************************
+
+**Recommended method for generating a single configuration**
+
+This function generates the IP and calls the parsing script.
+
+.. note::
+   This method works only with configurations where TX and RX have the same lane rate
+
+Call the get_diff_params method with the desired parameters.
+::
+   
+   get_diff_params 15.4 QPLL0 385 
+
+The first parameter represents the lane rate that will be set to both RX and TX. The 
+second one can be CPLL, QPLL0, QPLL1. The third one is the reference clock. If left 
+empty, then it will be filled with all the viable values for the lane rate given.
+::
+   
+   get_diff_params {9.8304} QPLL0 {} false 
+
+The fourth parameter is optional. If you set it to false, the script will remove from 
+the project and delete from disk the generated IPs after the list of parameters is done, 
+so you don't have to do that manually.
+
+Both the first and the third parameters are actually lists, so you can use that to 
+generate multiple configurations. Keep in mind that the script will generate IPs with 
+all the combinations between the lane rate and reference clock.
+::
+   
+   get_diff_params {9.8304 4.9152} QPLL0 {245.76 122.88} false 
+
+This call makes 4 instances of transceivers, and also deletes them after generating the 
+list because of the 4th parameter is set to false.
+
+Parsing script
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+If you used the get_diff_params method from the script to generate the IP, there is 
+no need to call it again.
+
+If you used the ad_gth_generator method from the script, you will have to call the 
+parsing script from the shell, as explained below.
+
+If you edited the IP in any way after generating it, make sure to generate output 
+products for the transceiver before going forward.
+
+Navigate to <project_name>.gen/sources_1/ip in the terminal. From there, call the 
+gtwiz_parser.pl script, specifying the GT type as in the example.
+::
+   
+   ../../../../../scripts/gtwiz_parser.pl GTHE4
+
+If you run the script wile having multiple configurations, it will include the unique 
+parameters for each IP, plus the gt_global list that contains the common parameters 
+within generated configurations that are different from the default values. Now, 
+you should find the files at <project_name>.gen/sources_1/ip Make sure to overwrite the 
+list from system_bd with the one in <GT_Type>_cfng.txt.
+
+.. warning::
+   
+   Please note that if you used the GUI method to instantiate the wizard, the paring script will not work
```

Comment:
Please revise the entire doc - try to wrap the text lines to 80 characters, if possible.